ql/src/java/org/apache/hadoop/hive/ql/metadata/Hive.java

```diff
@@ -875,6 +888,7 @@ public void alterPartition(String tblName, Partition newPart,
   public void alterPartition(String catName, String dbName, String tblName, Partition newPart,
                              EnvironmentContext environmentContext, boolean transactional)
       throws InvalidOperationException, HiveException {
+
```

Comment:
Please remove formatting changes - will not comment on the further ones :)

Reply:
Rebased with master and only kept relevant changes.